src/DssSpell.sol

```diff
@@ -79,112 +69,139 @@ contract DssSpellAction is DssAction {
+    address internal constant NAVIGATOR    = 0x11406a9CC2e37425F15f920F494A51133ac93072;
+    address internal constant QGOV         = 0xB0524D8707F76c681901b782372EbeD2d4bA28a6;
+    address internal constant UPMAKER      = 0xbB819DF169670DC71A16F58F55956FE642cc6BcD;
+    address internal constant PALC         = 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A;
+    address internal constant PBG          = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
+    address internal constant CLOAKY       = 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818;
+    address internal constant WBC          = 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47;
+    address internal constant BLUE         = 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf;
+
+    //  ---------- Math ----------
+    uint256 internal constant MILLION = 10 ** 6;
+
+    // ---------- Spark Proxy ----------
+    // Spark Proxy: https://github.com/marsfoundation/sparklend/blob/d42587ba36523dcff24a4c827dc29ab71cd0808b/script/output/1/primary-sce-latest.json#L2
+    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
 
     // ---------- Trigger Spark Proxy Spell ----------
-    address internal constant SPARK_SPELL                     = 0xFBdB6C5596Fc958B432Bf1c99268C72B1515DFf0;
+    address internal constant SPARK_SPELL = 0x95bcf659653d2E0b44851232d61F6F9d2e933fB1;
 
     function actions() public override {
-        // ---------- Management of ConsolFreight (RWA003-A) Default ----------
-        // Forum: https://forum.makerdao.com/t/consolfreight-rwa-003-cf4-drop-default/21745
+        // ---------- Stability Scope Parameter Changes ----------
+        // MIP: https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969
+        // Increase the ETH-A Stability Fee (SF) by 0.12% from 3.58% to 3.70%.
+        DssExecLib.setIlkStabilityFee("ETH-A", THREE_PT_SEVEN_ZERO_PCT_RATE, /* doDrip = */ true);
 
-        // Set DC to 0
-        // Note: it was agreed with GovAlpha that there will be no global DC reduction this time.
-        DssExecLib.setIlkDebtCeiling("RWA003-A", 0);
-        // Call tell() on RWALiquidationOracle
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).tell("RWA003-A");
+        // Increase the ETH-B Stability Fee (SF) by 0.12% from 4.08% to 4.20%.
+        DssExecLib.setIlkStabilityFee("ETH-B", FOUR_PT_TWO_ZERO_PCT_RATE, /* doDrip = */ true);
 
-        // ---------- Auth ESM on the Vow ----------
-        // Forum: http://forum.makerdao.com/t/overlooked-vectors-for-post-shutdown-governance-attacks-postmortem/20696/5
+        // Increase the ETH-C Stability Fee (SF) by 0.12% from 3.33% to 3.45%.
+        DssExecLib.setIlkStabilityFee("ETH-C", THREE_PT_FOUR_FIVE_PCT_RATE, /* doDrip = */ true);
 
-        DssExecLib.authorize(MCD_VOW, MCD_ESM);
+        // Activate DC-IAM for PSM-PAX-A
```

Comment:
Should be a proper section separator comment.
```suggestion
        // ---------- Activate DC-IAM for PSM-PAX-A ----------
```

Reply:
I think it's a subsection, not section (and it doesn't even have polls/forum links)

Reply:
Oh, okay... makes sense.